code/datums/components/food/edible.dm

```diff
@@ -452,6 +505,8 @@ Behavior that's still missing from this component that original food items had t
 			food_taste_reaction = FOOD_DISLIKED
 		else if(foodtypes & H.dna.species.liked_food)
 			food_taste_reaction = FOOD_LIKED
+	if(food_flags & FOOD_SILVER_SPAWNED) // it's not real food
```

Comment:
Remove this, it makes silver spawned food entirely inedible. I think it's out of scope to discuss it in this pr, and it's a blocking change as is.